pkgs/development/python-modules/slack-sdk/default.nix

```diff
@@ -1,35 +1,38 @@
 {
   lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+
+  # build-system
+  setuptools,
+
+  # dependencies
   aiodns,
   aiohttp,
   boto3,
-  buildPythonPackage,
-  fetchFromGitHub,
+  sqlalchemy,
+  websocket-client,
+  websockets,
+
+  # tests
   flake8,
   flask-sockets,
   moto,
-  pythonOlder,
   psutil,
   pytest-asyncio,
   pytestCheckHook,
-  setuptools,
-  sqlalchemy,
-  websocket-client,
-  websockets,
 }:
```

Comment:
Please revert this. If one wants to grouping the inputs for a second time I'm fine with that but for the modules I maintain I prefer to keep it aligned with the style of the majority of available Python modules in nixpkgs.

Reply:
Sure ! Done.